pkg/coreos/actuator.go

```diff
@@ -29,11 +30,14 @@ type actuator struct {
 	client client.Client
 	scheme *runtime.Scheme
 	logger logr.Logger
+	box    *packr.Box
 }
 
+//go:generate packr2
+
 // NewActuator creates a new Actuator that updates the status of the handled OperatingSystemConfigs.
 func NewActuator() operatingsystemconfig.Actuator {
-	return &actuator{logger: log.Log.WithName("coreos-operatingsystemconfig-actuator")}
+	return &actuator{logger: log.Log.WithName("coreos-operatingsystemconfig-actuator"), box: packr.New("templates", "./templates")}
```

Comment:
Why do we have to store these data in separate files? Can't we simply inject the content here? Then we don't need packr/box etc. WDYT?

Reply:
Done